javascript/src/test/pages/Admin/Admin.test.js

```diff
@@ -1,24 +1,25 @@
 import React from "react";
-import { render } from "@testing-library/react";
+import { render, waitFor } from "@testing-library/react";
+import userEvent from "@testing-library/user-event";
 import Admin from "main/pages/Admin/Admin";
 import { useAuth0 } from "@auth0/auth0-react";
 jest.mock("@auth0/auth0-react");
 import useSWR from "swr";
 jest.mock("swr");
+import { fetchWithToken } from "main/utils/fetch";
+jest.mock("main/utils/fetch");
 
 describe("Admin tests", () => {
   const admins = [
     {
       id: 2,
       email: "biden@usa.gov",
-      firstName: "Joe",
-      lastName: "Biden"
+      isPermanentAdmin: true
```

Comment:
It might be good to avoid anything that could be perceived as political in our code examples.

Reply:
Good point.